Description: Fix some POD errors reported by lintian and present
 in manpage for App::Daemon.
Origin: vendor
Bug: http://rt.cpan.org/Public/Bug/Display.html?id=51177
Forwarded: yes
Author: Salvatore Bonaccorso <salvatore.bonaccorso@gmail.com>
--- a/Daemon.pm
+++ b/Daemon.pm
@@ -485,6 +485,8 @@
     Running:     no
     Name match:  0
 
+=back
+
 =head2 Command Line Options
 
 =over 4
@@ -518,6 +520,8 @@
 option will be ignored if Log4perl is initialized independently or if
 a user-provided Log4perl configuration file is used.
 
+=back
+
 =head2 Setting Parameters
 
 Instead of setting paramteters like the logfile, the pidfile etc. from
@@ -597,8 +601,6 @@
 continue to run the code following the detach() call but return to the
 shell prompt immediately.
 
-=back
-
 =head1 AUTHOR
 
 Mike Schilli, cpan@perlmeister.com
